cdap-storage-ext-spanner/src/test/java/io/cdap/cdap/storage/spanner/SpannerStructuredTableAdminTest.java

```diff
@@ -103,17 +104,36 @@ public void testAdmin() throws Exception {
     admin.createOrUpdate(SIMPLE_TABLE_SPEC);
     Assert.assertTrue(admin.exists(SIMPLE_TABLE));
 
-    // Assert SIMPLE_TABLE schema
-    // ONLY checking compatibility because of INT/LONG to INT64 conversion in Spanner
+    // Assert SIMPLE_TABLE schema: checking equality after compatible conversion because of INT/LONG
+    // to INT64 conversion in Spanner
     StructuredTableSchema simpleTableSchema = admin.getSchema(SIMPLE_TABLE);
-    Assert.assertTrue(simpleTableSchema.isCompatible(SIMPLE_TABLE_SPEC));
+    Assert.assertEquals(simpleTableSchema, convertSpecToCompatibleSchema(SIMPLE_TABLE_SPEC));
 
     // Update SIMPLE_TABLE spec to UPDATED_SIMPLE_TABLE_SPEC
     admin.createOrUpdate(UPDATED_SIMPLE_TABLE_SPEC);
 
     // Assert UPDATED_SIMPLE_TABLE_SPEC schema
     StructuredTableSchema updateSimpleTableSchema = admin.getSchema(SIMPLE_TABLE);
-    Assert.assertTrue(updateSimpleTableSchema.isCompatible(UPDATED_SIMPLE_TABLE_SPEC));
+    Assert.assertEquals(
+        updateSimpleTableSchema, convertSpecToCompatibleSchema(UPDATED_SIMPLE_TABLE_SPEC));
+  }
+
+  @Test
+  @Override
+  public void testCreateOrUpdateTwiceShouldSucceed() throws Exception {
+    StructuredTableAdmin admin = getStructuredTableAdmin();
+
+    // Assert SIMPLE_TABLE Empty
+    Assert.assertFalse(admin.exists(SIMPLE_TABLE));
+
+    // Call createOrUpdate SIMPLE_TABLE twice
```

Comment:
nit: add comment on why you call it twice.

Reply:
SG, added the comments